jpo-geojsonconverter/pom.xml

```diff
@@ -130,12 +130,12 @@
         <dependency>
             <groupId>usdot.jpo.ode</groupId>
             <artifactId>jpo-ode-core</artifactId>
-            <version>3.0.0</version>
+            <version>4.0.0</version>
         </dependency>
         <dependency>
             <groupId>usdot.jpo.ode</groupId>
             <artifactId>jpo-ode-plugins</artifactId>
-            <version>3.0.0</version>
+            <version>4.0.0</version>
```

Comment:
non-blocking: Can we pull the ODE version out to the "properties" section?

Reply:
Addressed